### ISSUE TITLE
New closeBracket options throw an error

### ISSUE THREAD

**Issue by leosdad** [vuejs/eslint-plugin-vue#1220]
* **ESLint version:** 7.3.1
* **eslint-plugin-vue version:** 6.2.2
* **Node version:** 12.18.1

**Please show your full configuration:**

```json
{
    "env": {
        "browser": true,
        "es6": true
    },
    "extends": [
        "plugin:vue/recommended",
        "standard"
    ],
    "globals": {
        "Atomics": "readonly",
        "SharedArrayBuffer": "readonly"
    },
    "parserOptions": {
        "ecmaVersion": 2018,
        "sourceType": "module"
    },
    "plugins": [
        "vue"
    ],
    "rules": {
		"vue/html-indent": [ "warn", "tab", {
			"attribute": 1,
			"baseIndent": 1,
			"closeBracket": {
				"startTag": 1,
				"endTag": 2,
				"selfClosingTag": 1
			}
		}],
		"vue/html-quotes": "error",
		"vue/multiline-html-element-content-newline": "off",
		"vue/no-unused-components": "warn",
		"vue/no-unused-vars": "warn",
		"vue/no-v-html": "off",
		"vue/require-prop-type-constructor": "off",
		"vue/singleline-html-element-content-newline": "off",

		"array-bracket-spacing": "warn",
		"block-spacing": "warn",
		"brace-style": "off",
		"comma-dangle": "off",
		"comma-spacing": "warn",
		"curly": ["error"],
		"eqeqeq": ["error", "always"],
		"indent": "off",
		"no-console": "warn",
		"no-magic-numbers": ["warn", { "ignore": [0, 1, -1, 2], "ignoreArrayIndexes": true }],
		"no-mixed-spaces-and-tabs":"warn",
		"no-multiple-empty-lines": ["warn", {"max": 1} ],
		"no-prototype-builtins":"warn",
		"no-tabs": "off",
		"no-trailing-spaces": ["warn", { "ignoreComments": true }],
		"no-unused-vars": "warn",
		"object-curly-newline": "warn",
		"object-curly-spacing": "warn",
		"object-property-newline": "warn",
		"operator-linebreak": ["warn", "after"],
		"padded-blocks": "off",
		"quotes": "warn",
		"semi": ["error", "always"],
		"space-before-function-paren": ["warn", "never"],
		"space-in-parens": "warn",
		"spaced-comment": "warn",
		"template-curly-spacing": "warn",
		"keyword-spacing": ["warn", { "overrides": {
			"if": { "after": false },
			"for": { "after": false },
			"do": { "after": false },
			"while": { "after": false },
			"switch": { "after": false }
		  } }]
    }
}
```

**What did you do?**	

```vue
	<div
		class="flex-center full-width q-gutter-sm row"
		>
		<q-icon
			size="2em"
			:name="icon"
			:color="icon"
		/>
		<span>{{ message }}</span>
	</div>
```

**What did you expect to happen?**

I want this snipped to be validated by the `vue/html-indent` rule with no errors.

**What actually happened?**

I got the following error instead:
```
at c:\Users\Rubem Pechansky\.vscode\extensions\dbaeumer.vscode-eslint-2.1.5\server\out\eslintServer.js:1:36902
    at ce (c:\Users\Rubem Pechansky\.vscode\extensions\dbaeumer.vscode-eslint-2.1.5\server\out\eslintServer.js:1:46186)
(node:22024) UnhandledPromiseRejectionWarning: Error: web\.eslintrc.json:
	Configuration for rule "vue/html-indent" is invalid:
	Value {"startTag":1,"endTag":2,"selfClosingTag":1} should be integer.
```

**Comment by ota-meshi** [comment]
Thank you for this issue.

That option is available in v7.0.0.alpha.  Not available in v6.x.

**Comment by leosdad** [comment]
Thanks! I'll keep an eye on this package then.
